psamm/metabolicmodel.py

```diff
@@ -17,6 +17,8 @@
 
 """Representation of metabolic network models."""
 
+from __future__ import unicode_literals
+
```

Comment:
This module has a `__repr__` method on `FluxBounds` which needs to be corrected so that it returns `str` type in Python 2.7.